### PR TITLE
Handle the no-discord-API-token case

### PR DESCRIPTION
**Pull request by dan-simon**
What it says. I haven't added tests or checked very hard that this doesn't break other stuff and honestly what I did seemed a bit too simple.

Just to be clear, this PR is meant to, if no discord API token is in config, just not create discord channels (but otherwise do the same things).

### CODE REVIEW

**Review by asdfryan — APPROVED** [review]

**Review by rgossiaux — CHANGES_REQUESTED** [review]
What's the prod_db.sql file doing there?

**Comment by dan-simon** [comment]
I think it wasn't caught by .gitignore. I'll remove it.

File: hunts/views.py [hunk 1]
```diff
@@ -173,10 +173,14 @@ def post(self, request, hunt_slug):
             if google_api_client:
                 google_api_client.add_puzzle_link_to_sheet(puzzle_url, sheet)
 
-            chat_room = ChatRoom.objects.create(
-                service=settings.CHAT_DEFAULT_SERVICE, name=name
-            )
-            chat_room.create_channels()
+            if settings.CHAT_DEFAULT_SERVICE is not None:
```
Comment (akirabaruah) [inline hunts/views.py:176]: nit: Can we simplify this to just `if settings.CHAT_DEFAULT_SERVICE:`? It's a bit nicer to read and also catches the empty string case.

**Review by akirabaruah — APPROVED** [review]
LGTM with a nit - thanks Dan!

**Review by rgossiaux — APPROVED** [review]
Thanks!

**Pull request merged**